src/backend/app/tasks/task_routes.py

```diff
@@ -86,3 +89,43 @@ async def new_event(
         user_data,
         background_tasks,
     )
+
+
+@router.post("/upload/{project_id}/{task_id}")
```

Comment:
This probably isn't a comment specific to this PR, but the project as a whole.

The endpoint structure for FMTM doesn't necessarily conform to the REST spec / best practices, but we are working on it & hopefully that's changing over time 🙏 

So typically it would be set up like this:

If the task id is a unique identifier on it's own.
`/projects/{project_id}`
`/tasks/{task_id}`

If task ids are only unique in relation to the project they are within:
`/projects/{project_id}`
`/projects/{project_id}/tasks/{task_id}`

So for this endpoint, as task_id is a globally unique UUID, it would make sense to create:

`/tasks/{task_id}/upload-ortho`

or something similar.

Just for info going forward! I'm not saying things have to change immediately, or even at all, but it could be a nice to have into the future 😄